src/components/overview/OverviewMain.jsx

```diff
@@ -69,6 +76,70 @@ class overviewMain extends React.Component {
     }
   }
 
+  //this function is attached to the thumbnail images in the image carousel and
+  //changes the selected image when these images are clicked.
+  //My logic is needlessly complex--I'm using an index for my current image when I should be
+  //using the url. Will change perhaps in the future
+  changeImageOnThumbnailClick(newThumbnailUrl) {
+    for (let i = 0; i < this.state.images.otherImagesInStyle.length; i++) {
+      if (
+        this.state.images.otherImagesInStyle[i].thumbnail_url ===
+        newThumbnailUrl
+      ) {
+        this.setState({
+          images: {
+            currentImage: i,
```

Comment:
Does this refer to the index of the current image? Maybe clarify in variable name?